commands/client-list.md

```diff
@@ -1,7 +1,7 @@
-The `CLIENT LIST` command returns information and statistics about the client
+The `CLIENT LIST [TYPE type]` command returns information and statistics about the client
```

Comment:
@itamarhaber will that change cause the link to the client list page to stop working?

Reply:
Yep, but that's not terrible as the user is already on that page. However, I don't think the syntax needs to be repeated here, so this should be reverted.

Reply:
ok